packages/p2p/src/stores/general-store.js

```diff
@@ -204,6 +204,9 @@ export default class GeneralStore extends BaseStore {
     showCompletedOrderNotification(advertiser_name, order_id) {
         const notification_key = `order-${order_id}`;
 
+        // we need to refresh notifications in notifications-store in the case of a bug where when user closes the notification, the notification count is not synced up with the closed notification
```

Comment:
Add new line for better readability
```suggestion
        // we need to refresh notifications in notifications-store in the case of a bug when user closes the notification, the notification count is not synced up with the closed notification
```

Reply:
Got it!